infrastructure/main.tf

```diff
@@ -102,4 +107,19 @@ resource "azurerm_key_vault_secret" "staging_db_database" {
   value        = module.reform-notifications-staging-db.postgresql_database
 }
 
-# endregion
\ No newline at end of file
+# endregion
+
+# region Copy secrets from BulkScan
+
+data "azurerm_key_vault_secret" "bulk_scan_processor_s2s_secret" {
+  key_vault_id = "${data.azurerm_key_vault.bulk_scan_key_vault.id}"
+  name         = "s2s-secret"
+}
+
+resource "azurerm_key_vault_secret" "test_bsp_s2s_secret" {
+  key_vault_id = data.azurerm_key_vault.reform_scan_key_vault.id
+  name         = "test-bsp-s2s-secret"
+  value        = data.azurerm_key_vault_secret.bulk_scan_processor_s2s_secret.value
+}
```

Comment:
How about just test-s2s-secret? All we care about is that it's an s2s secret for tests.

Maybe we could also create a pair of secrets in s2s (microservice key and name) and use those from s2s key vault?

Reply:
well, it's a different resource group. there's not harm in having extra prefix level

Reply:
It's a bit confusing to have a secret named "test bulk scan processor's s2s secret" in reform scan notification service, which itself is a part of a different product and is unaware of bulk scan processor.